### PR TITLE
Drop server 4.0 from integration test matrix

### PR DESCRIPTION
**Pull request by robsdedude**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -98,7 +98,6 @@ def generate_tc_config(version, enterprise, cluster, scheme, stress_test):
             ("3.5",    True,        False,    "bolt",   0),
             ("3.5",    True,        True,     "neo4j", 60),
             # not officially supported versions
-            ("4.0",    True,        False,    "neo4j",  0),
```
Comment (fbiville) [inline main.py:101]: this removes only 4.0, not 4.1 (which should be removed as well)

**Review by fbiville — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (robsdedude) [reply main.py:101]: I don't think so. 4.4 drivers were built to work with 4.1 servers and should still be tested against them.

EDIT: Also, I suggested to have a feature freeze on TestKit 4.4. 4.0 ITs shouldn't have been there from the start while 4.1 should have.

Comment (fbiville) [reply main.py:101]: I'm not sure I'm following the reasoning: why would we remove 4.0 but not 4.1? Both reached [EOL](https://neo4j.com/developer/kb/neo4j-supported-versions/).

Comment (fbiville) [reply main.py:101]: needs some more internal decision first for 4.1. Let's consider this resolved for now.

**Review by fbiville — APPROVED** [review]

**Pull request merged**